### PR TITLE
Fix/pdf report

### PR DESCRIPTION
**Pull request by AlviseSembenico**

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -28,7 +28,7 @@ django-ordered-model<3.0
 diff-match-patch==20121119
 redis>=2.10.0,<3
 django-background-tasks
-WeasyPrint
+django-wasyprint
```
Comment (alextreme) [inline requirements/base.in]: wasyprint? that's new for me

**Comment by alextreme** [comment]
Perhaps we need to work on this some more if we want a more pretty report like what postman gives us.

How about sitting together with Sven to see if you can get closer to the style you want using regular HTML, and then seeing if weasyprint or chrome --print (like Joeri and Alexander use for Interflow) is a better option?

File: requirements/base.in (hunk 1 above)
Comment (AlviseSembenico) [reply requirements/base.in]: i'd like to say that it is a new version but it was my mistake

**Comment by AlviseSembenico** [comment]
Yes, it may be the best option

**Pull request closed without merging**